src/table/table.stories.ts

```diff
@@ -29,6 +31,49 @@ import { clone } from "../utils/utils";
 
 const en = require("./../../src/i18n/en.json");
 
+@Component({
+	selector: "app-table",
+	template: `
+		<ibm-table
+			[model]="model"
+			[size]="size"
+			[showSelectionColumn]="showSelectionColumn"
+			[striped]="striped"
+			(sort)="simpleSort($event)">
+		</ibm-table>
+	`
+})
+class TableStory implements OnInit, OnChanges {
+	@Input() model = new TableModel();
+	@Input() size = "md";
+	@Input() showSelectionColumn = true;
+	@Input() striped = true;
+	@Input() sortable = true;
+	@Input() simpleSort;
```

Comment:
Since it doesn't seem feasible to conditionally provide an observer in template, does it still make sense to provide the sorting function as an input?

Reply:
I think it still makes sense because the user still needs to see the sorting in action (which is enabled by default in the story)

Reply:
Sure, but it can be implemented in a TableStory instead provided to it from the outside.
That's the way devs would do it anyway.

Let's change it to that way for the people who copy code from the demos.

_The code from before didn't take a function as an input, it was added as an observer to `(sort)`, which we can no longer simply do because `ibm-table` is no longer directly exposed, so `simpleSort()` needs to move from global to the `TableStory` component (and doesn't need to be a prop)._